documentation/src/docs/streamcuts.md

```diff
@@ -92,4 +92,4 @@ It must be noted that passing ```StreamCut.UNBOUNDED``` to startStreamCut and en
 current head of stream and the current tail of the stream, respectively.
 
 
-We have provided a simple yet illustrative example of using StreamCut [here.](https://github.com/pravega/pravega-samples/tree/{{variable.pravega_samples_verion}}/pravega-client-examples/src/main/java/io/pravega/example/streamcuts)
+We have provided a simple yet illustrative example of using StreamCut [here.](https://github.com/pravega/pravega-samples/tree/v0.3.1/pravega-client-examples/src/main/java/io/pravega/example/streamcuts)
```

Comment:
Why this change?

Reply:
otherwise the link throws 404 error

Reply:
This doesn't sound like the right way of solving this. I believe @Prabha-Veerubhotla added that variable (`variable.pravega_samples_verion`) there so that we didn't have to update the link upon every new version. Where is that variable supposed to be updated?

Note that there is a typo in the variable name, so it is possible that the link is broken because of the typo.

Reply:
ok will check

Reply:
version passing through variable is not fixed by prabha